### DISCUSSION TITLE
No. of connections

### DISCUSSION THREAD

**Discussion by email2vimalraj** [vectordotdev/vector#17839] [Q&A] [answered]
I'm evaluating Vector as an agent and using Splunk HEC as a Sink.

I would like to limit the number of connections between the source server to Splunk HEC endpoint due to firewall in between. Don't want to cause any DDoS on Firewall due to limitless connections to flush the data. What are the possibilities to limit?

Also, as mentioned in the `internal_metrics` source documentation, I don't see any `connection` metrics populated. Are there any specific configurations to be enable to populate `connection` metrics?

**Comment by jszwedko** [comment, marked as answer]
Hi @email2vimalraj !

Unfortunately I'm not seeing any metric for that. Would you want to open a feature request? We'd also accept a feature request to limit the number of open connections for sinks.

I think Vector should automatically deal with the firewall limiting the connections by retrying the requests though.
